Integrations/integration-MISP_V2.yml

```diff
@@ -1642,7 +1678,23 @@ script:
       description: All tag names in the event
       type: string
     description: Adds a tag to the given UUID event or attribute.
+  - name: misp-add-sighting
+    arguments:
+    - name: type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - sighting
+      - false-positive
+      - expiration
+      description: Type of sighting to add
+    - name: id
+      description: ID of attribute to add sighting to
+    - name: uuid
+      description: UUID of attribute to add sighting to
```

Comment:
Please add to the description where this `UUID` can be retrieved from, like from which command.